app/javascript/src/component_branch.js

```diff
@@ -330,7 +330,9 @@ class BranchAnswer {
       $answer.val([]);
     } else {
       $answer.show();
-      $answer.val( $answer.find('option:first').val() );
+      if( !$answer.val() ) {
```

Comment:
Wondering if it would have been easier to do and understand intention if something like...
```
    if(hideAnswers) {
      $answer.hide();
      $answer.attr("disabled", true);
    }
    else {
      $answer.attr("disabled", false);
      $answer.show();
    }
```
...but not enough for me to want to hold up these changes.